src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersByOrganisationIntegrationTest.java

```diff
@@ -180,7 +209,7 @@ private void validateUsers(Map<String, Object> response, int expectedUserCount)
             assertThat(user.get("firstName")).isNotNull();
             assertThat(user.get("lastName")).isNotNull();
             assertThat(user.get("email")).isNotNull();
-            if (user.get("idamStatus").equals(IdamStatus.ACTIVE.toString())) {
+            if (user.get("idamStatus").equals(IdamStatus.ACTIVE.toString()) && rolesReturned) {
                 assertThat(((List) user.get("roles")).size()).isEqualTo(1);
```

Comment:
instead of checking for size only, you should also assert the value returned

Reply:
I will be able to check using:
`assertThat(((List) user.get("roles")).contains("pui-"));`
However, I cannot assert full role names as they change and vary each time the test case is ran